nc-news-bh-final-vite/src/utils/api.js

```diff
@@ -37,4 +37,32 @@ function fetchComments(article_id){
     })
 }
 
-export {fetchArticles, fetchArticle, fetchComments}
\ No newline at end of file
+function upVote(article_id){
+    return ncNewsAPI
+    .patch(`/articles/${article_id}`, {inc_votes: 1,})
+    .then((res)=>{
+        return res.data
+    })
+    .catch((err)=>{
+        console.log(err)
+    })
+}
+
+function downVote(article_id){
+    return ncNewsAPI
+    .patch(`/articles/${article_id}`, {inc_votes: -1,})
+    .then((res)=>{
+        return res.data
+    })
+    .catch((err)=>{
+        console.log(err)
+    })
+}
```

Comment:
We should really only have one function making the patch API call. Could we refactor these 2 into 1 please?